src/enums/nameRequestStates.ts

```diff
@@ -12,6 +12,8 @@ export enum NameRequestStates {
   INPROGRESS = 'INPROGRESS',
   REJECTED = 'REJECTED',
   NRO_UPDATING = 'NRO_UPDATING',
+  NO_CONTACT = 'NO_CONTACT',
+  INCORRECT_CONTACT = 'INCORRECT_CONTACT',
```

Comment:
I believe lines 26 and 27 are now obsolete. Please delete them if they're not used anywhere else. (You may have to update unit tests.)

Reply:
I updated

Reply:
Are there no more references to the old enums, even in comments?